environment.yml

```diff
@@ -0,0 +1,11 @@
+name: aTRAM
+channels:
+  - bioconda
+  - defaults
+dependencies:
+  - blast
+  - velvet
+  - trinity
+  - abyss
+  - spades
+  - exonerate
```

Comment:
The only way I could get this was by adding:
`  - conda-forge` to the channels section and to add
```
  - biopython
  - numpy
  - psutil
```
to the dependencies section.

Reply:
this `environment.yml` file assumes that you have previously installed the `aTRAM` package lines above, that's why these packages:
```
 - biopython
  - numpy
  - psutil
```
will not be installed when you just use the `environment.yml` file. However, this file has been updated and it currently installs `aTRAM` directly. To do so, it relies on the repo address:
```
- pip:
    - git+https://github.com/Ulises-Rosas/aTRAM.git#egg=atram[complete]
```
Please feel free to change this repo address when pathways of scrips in the `setup.py` file are updated